powersimdata/design/investment/investment_costs.py

```diff
@@ -59,27 +84,35 @@ def select_mw(x, cost_df):
         :return: (*pandas.Series*) -- series of [*'MW'*, *'costMWmi'*] to be assigned
             to branch.
         """
-
+        underground_regions = ("NEISO", "NYISO J-K")
+        cost_df = cost_df.copy()
```

Comment:
I understand the purpose here, however, it would be better to use a different variable name to reduce confusion.

Reply:
Done.